webserver/webserver/src/main/java/io/helidon/webserver/NettyWebServer.java

```diff
@@ -414,6 +417,66 @@ public int port(String name) {
         return address instanceof InetSocketAddress ? ((InetSocketAddress) address).getPort() : -1;
     }
 
+    private Transport transport() {
+        Transport transport = configuration.transport().orElse(new NioTransport());
+        return transport.isAvailableFor(this) ? transport : new NioTransport();
+    }
+
+    @SuppressWarnings("unchecked")
+    private EventLoopGroup bossGroup() {
+        return transport().createTransportArtifact(EventLoopGroup.class, "bossGroup", configuration).get();
```

Comment:
Why does the method return an optional, if you call `.get` on it? If this is mandatory, it should not return optional. If it is optional, you should check for the existence of value and throw a meaningful exception